src/LogicTerm/TermImpl.cpp

```diff
@@ -40,9 +40,9 @@ TermImpl::TermImpl(const TermImpl& other):
     nodes.insert(nodes.end(), other.getNodes().begin(), other.getNodes().end());
 }
 
-std::string TermImpl::getStrRep(OpType opType) {
+std::string TermImpl::getStrRep(OpType type) {

```

Comment:
Inconsistent change of parameter, probably should be op?
```suggestion
std::string TermImpl::getStrRep(OpType op) {

```

Reply:
Probably, yes 👍🏻